kafka/conn.py

```diff
@@ -79,7 +79,7 @@ def _raise_connection_error(self):
 
     def _read_bytes(self, num_bytes):
         bytes_left = num_bytes
-        responses = []
+        result_buffer = six.BytesIO()
```

Comment:
is `six` needed here?  why not just use `io.BytesIO` which I believe is supported as of python 2.6 ?